modules/patterns/pages/gitops-for-manual-releases.adoc

```diff
@@ -0,0 +1,119 @@
+
+For teams with that posture, we recommend using a gitops process to manage release decisions. Treat your releases as code and gain the benefits of code review and multi-party sign-off.
+
+This pattern has the additional benefit of helping you manage a race condition with xref:testing:integration/snapshots/index.adoc[snapshot garbage collection]. Without this pattern, a slower-moving team may run into a situation where a snapshot that they decide to release is garbage-collected from the cluster before they can actually release it. The pattern here will guide you to export your release candidate snapshot to git so that you can be sure it is available and unchanged at release time.
+
+== Manage Manual Releases with a GitOps process
+
+.Setup
+
+. For your team, create a git repository in an SCM system of your choice (like GitHub, GitLab, or Forgejo).
+. Configure the SCM to require approval from the people on your team who should be involved in approving a release. Use of `CODEOWNERS` is supported in both link:https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners[github] and link:https://docs.gitlab.com/user/project/codeowners/[gitlab]. With the following configuration, one person from all four groups will be required to sign off in order for a change to be merged to this repository.
+
+[source]
+----
+❯ cat CODEOWNERS
+[Managers]
+* @sally
+
+[Devs]
+* @jen @soren
+
+[QE]
+* @remus
+
+[Docs]
+* @john
+----
+
+
+. Establish a directory structure like the following:
+
++
+[source]
+----
+❯ tree
+.
+└── releases
+    ├── 1.2.0
+    │   ├── release.yaml
+    │   └── snapshot.yaml
+    └── 1.2.1
+        ├── release.yaml
+        └── snapshot.yaml
+----
+
+. Optional: set up an in-SCM pipeline (like a github workflow or a gitlab-ci pipeline) that will `kubectl apply -f releases/$RELEASE`, but only for new files that change. If you decide to do this, you'll need to procure a serviceaccount with the link:https://github.com/redhat-appstudio/infra-deployments/blob/main/components/konflux-rbac/production/base/konflux-releaser-bot-actions.yaml[permissions to create Releases and Snapshots] in your tenant namespace. Without this, you can manually `kubectl apply -f releases/$RELEASE` after merge. Manually doing this for your first few releases is appropriate while your team gets used to this new workflow.
+
+.Procedure
+
+Let's say you're preparing for an upcoming 1.3.0 release.
+
+. Create a new git branch for this:
+
++
+[source]
+----
+❯ git checkout -b release-1.3.0
+----
+
+. Create the directory: `mkdir releases/1.3.0`
+
++
+[source]
+----
+❯ mkdir releases/1.3.0
+----
+
+. Find the `Snapshot` that you intend to release, and export it from your namespace with:
+
++
+[source]
+----
+❯ kubectl get snapshots snapshot-7mb5s -o yaml | kubectl neat > releases/1.3.0/snapshot.yaml
```

Comment:
not sure most ppl have this plugin installed... I didn't. maybe mention it as a note?